app/assets/javascripts/vendor/jquery.tipsy.js

```diff
@@ -1,337 +1 @@
-      var pointee = $.data(this, 'tipsy-pointee');
-      if (!pointee || !isElementInDOM(pointee)) {
-        $(this).remove();
-      }
-    });
-  };
-
-  $.fn.tipsy.enable = function() {
-    $.fn.tipsy.enabled = true;
-  };
-
-  $.fn.tipsy.disable = function() {
-    $.fn.tipsy.enabled = false;
-  };
-
-  // Overwrite this method to provide options on a per-element basis.
-  // For example, you could store the gravity in a 'tipsy-gravity' attribute:
-  // return $.extend({}, options, {gravity: $(ele).prop('tipsy-gravity') || 'n' });
-  // (remember - do not modify 'options' in place!)
-  $.fn.tipsy.elementOptions = function(ele, options) {
-    return $.metadata ? $.extend({}, options, $(ele).metadata()) : options;
-  };
-
-  $.fn.tipsy.autoNS = function() {
-    return $(this).offset().top > ($(document).scrollTop() + $(window).height() / 2) ? 's' : 'n';
-  };
-
-  $.fn.tipsy.autoWE = function() {
-    return $(this).offset().left > ($(document).scrollLeft() + $(window).width() / 2) ? 'e' : 'w';
-  };
-
-  $.fn.tipsy.autoNWNE = function() {
-    return $(this).offset().left > ($(document).scrollLeft() + $(window).width() / 2) ? 'ne' : 'nw';
-  };
-
-  $.fn.tipsy.autoSWSE = function() {
-    return $(this).offset().left > ($(document).scrollLeft() + $(window).width() / 2) ? 'se' : 'sw';
-  };
-
-  /**
-   * yields a closure of the supplied parameters, producing a function that takes
-   * no arguments and is suitable for use as an autogravity function like so:
-   *
-   * @param margin (int) - distance from the viewable region edge that an
-   *        element should be before setting its tooltip's gravity to be away
-   *        from that edge.
-   * @param prefer (string, e.g. 'n', 'sw', 'w') - the direction to prefer
-   *        if there are no viewable region edges effecting the tooltip's
-   *        gravity. It will try to vary from this minimally, for example,
-   *        if 'sw' is preferred and an element is near the right viewable
-   *        region edge, but not the top edge, it will set the gravity for
-   *        that element's tooltip to be 'se', preserving the southern
-   *        component.
-   */
-  $.fn.tipsy.autoBounds = function(marginNorth, marginEast, prefer) {
-    return function() {
-      var dir = {ns: prefer[0], ew: (prefer.length > 1 ? prefer[1] : false)},
-        boundTop = $(document).scrollTop() + marginNorth,
-        boundLeft = $(document).scrollLeft() + marginEast,
-        $this = $(this);
-
-      if ($this.offset().top < boundTop) dir.ns = 'n';
-      if ($this.offset().left < boundLeft) dir.ew = 'w';
-      if ($(window).width() + $(document).scrollLeft() - $this.offset().left < marginEast) dir.ew = 'e';
-      if ($(window).height() + $(document).scrollTop() - $this.offset().top < marginNorth) dir.ns = 's';
-
-      return dir.ns + (dir.ew ? dir.ew : '');
-    };
-  };
-
-})(jQuery);
\ No newline at end of file
+// TODO: make sure this is not imported in any dev portal
```

Comment:
According to your last reply to Joan, maybe we can even remove this file?

Reply:
Ah yes, good catch!